sdk/include/xdk/setypes.h

```diff
@@ -1053,6 +1053,12 @@ typedef struct _TOKEN_ELEVATION {
   $ULONG TokenIsElevated;
 } TOKEN_ELEVATION, *PTOKEN_ELEVATION;
 
+typedef enum _TOKEN_ELEVATION_TYPE {
```

Comment:
I therefore wonder whether we need to surround this typedef by `#if (NTDDI_VERSION >= NTDDI_VISTA)  .... #endif`